maven/plugins/com.liferay.ide.maven.core/src/com/liferay/ide/maven/core/LiferayMavenProjectProvider.java

```diff
@@ -176,18 +178,34 @@ public IStatus doCreateNewProject(
             final List<?> archProps =
                 archetypeManager.getRequiredProperties( archetype, remoteArchetypeRepository, monitor );
 
-           if( ! CoreUtil.isNullOrEmpty( archProps ) )
-           {
-               for( Object prop : archProps )
-               {
-                   if( prop instanceof RequiredProperty )
-                   {
-                       final RequiredProperty rProp = (RequiredProperty) prop;
-
-                       properties.put( rProp.getKey(), rProp.getDefaultValue() );
-                   }
-               }
-           }
+            if( !CoreUtil.isNullOrEmpty( archProps ) )
+            {
+                for( Object prop : archProps )
+                {
+                    if( prop instanceof RequiredProperty )
+                    {
+                        final RequiredProperty rProp = (RequiredProperty) prop;
+
+                        final String key = rProp.getKey();
+
```

Comment:
I know its a small thing but lets not get the themeParent/Framework values unless they are needed.